### PR TITLE
Resolve some targets of examples and his doc

### PR DESCRIPTION
**Pull request by eduazocar**
### Contribution description
PR dedicated to replace some target boards from examples and change his documentation, this will be complement these sources

### CODE REVIEW

File: examples/moisture_sensor_hw390/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-APPLICATION = test_esp32
-BOARD ?= esp32-wrover-kit
+APPLICATION = Moisture_sensor_test
```
Comment (luisan00) [inline examples/moisture_sensor_hw390/Makefile:1]: no caps

File: examples/external_led/main.c [hunk 2]
```diff
@@ -15,9 +15,11 @@
  */
 
 /**
+ * @ingroup     examples_esp32-wrover
+ *
  * @file
- * @brief
- * @copyright Copyright (c) 2021
+ * @brief       this is an example of external_led_blink
+ * @author      xkevin190 <kevinvelasco190@gmail.com>
```
Comment (luisan00) [inline examples/external_led/main.c:22]:
```suggestion
 * @author      xkevin190 <kevinvelasco190@gmail.com>
 * @author      add here you nickname etc etc
```

File: examples/esp32-wrover-app/main.c [hunk 3]
```diff
@@ -15,11 +15,13 @@
  */
 
 /**
- * @brief       Main example file
+ * @ingroup     examples_esp32-wrover
  *
+ * @file
+ * @brief       A example app of ESP32-wrover
  * @author      xkevin190 <kevinvelasco190@gmail.com>
```
Comment (luisan00) [inline examples/esp32-wrover-app/main.c:22]:
```suggestion
 * @author      xkevin190 <kevinvelasco190@gmail.com>
 * @author      add your name here
```

File: examples/moisture_sensor_hw390/main.c [hunk 4]
```diff
@@ -15,9 +15,11 @@
  */
 
 /**
+ * @ingroup     examples_esp32-wrover
+ *
  * @file
- * @brief moisture sensor test
- * @author xkevin190 <kevinvelasco190@gmail.com>
+ * @brief       Moisture sensor test
+ * @author      xkevin190 <kevinvelasco190@gmail.com>
```
Comment (luisan00) [inline examples/moisture_sensor_hw390/main.c:22]: here your name too

**Review by luisan00 — CHANGES_REQUESTED** [review]
some dead kittens xD

File: examples/esp32-wroom-32/test_ds18b20/README.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+@ingroup d18b20_examples
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/README.md:1]: Looks better as follow
```suggestion
@ingroup d18b20
```

File: examples/esp32-wroom-32/slipdev/slipdev_params.h [hunk 6]
```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup examples_esp32-wroom
+ * @ingroup Slipdev_example
```
Comment (luisan00) [inline examples/esp32-wroom-32/slipdev/slipdev_params.h:10]:
```suggestion
 * @ingroup slipdev
```

File: examples/esp32-wroom-32/slipdev/README.md [hunk 7]
```diff
@@ -1,3 +1,7 @@
+/**
+ * @ingroup Slipdev_example README.md
```
Comment (luisan00) [inline examples/esp32-wroom-32/slipdev/README.md:2]:
```suggestion
 * @ingroup slipdev
```

File: examples/doc.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+/**
+@defgroup m4a-examples Mesh4all Examples
```
Comment (luisan00) [inline examples/doc.txt:2]:
```suggestion
@defgroup examples Mesh4all Examples
```

File: examples/esp32-wroom-32/slipdev/doc.txt [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+ * Copyright (C) 2022 Mesh4all <mesh4all.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    Slipdev_example Slipdev
+ * @ingroup     m4a-examples
```
Comment (luisan00) [inline examples/esp32-wroom-32/slipdev/doc.txt:11]:
```suggestion
 * @defgroup    slipdev
 * @ingroup     examples
```

File: examples/esp32-wroom-32/test_ds18b20/doc.txt [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+ * Copyright (C) 2022 Mesh4all <mesh4all.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    d18b20_examples d18b20 Example
+ * @ingroup     m4a-examples
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/doc.txt:11]:
```suggestion
 * @ingroup     examples
```

File: examples/doc.txt [hunk 11]
```diff
@@ -0,0 +1,7 @@
+/**
+@defgroup examples Mesh4all Examples
+
+## This is example-doc
```
Comment (luisan00) [inline examples/doc.txt:4]: this should be developed in the near future :+1:

**Review by luisan00 — CHANGES_REQUESTED** [review]
in the examples folder look at  `TPM_UART_APP` , the name should not  be upper case

File: examples/doc.txt (hunk 11 above)
Comment (eduazocar) [reply examples/doc.txt:4]: Do you refers to last file changed?

**Comment by eduazocar** [comment]
Do you refers to last file changed?

**Comment by eduazocar** [comment]
i didn't understand. I renamed the TPM_UART_APP folder to tmp_uart folder...

**Comment by luisan00** [comment]
Then all's good, well done ! Do a squash and ready to move forward @eduazocar

**Comment by eduazocar** [comment]
Ok everything it's ready

**Review by luisan00 — APPROVED** [review]

**Comment by luisan00** [comment]
> Ok everything it's already @luisan00

Let's go ahead :+1:

**Pull request merged**